sangwon/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUR PORT NUMBER HOULD BE HERE!
+
+TYPEORM_CONNECTION = mysql
```

어떤 
```suggestion
TYPEORM_CONNECTION = mysql // <- 중의적인 단어로 수정!
```

어떠한 rdbms를 쓰는지 직접적으로 작성하는것이 아닌 중의적인 단어로 수정해주세요~